tests/lib/rules/no-plusplus.js

```diff
@@ -26,14 +26,31 @@ ruleTester.run("no-plusplus", rule, {
         { code: "var foo = 0; foo=+1;", options: [{allowForLoopAfterthoughts: true}] },
         { code: "for (i = 0; i < l; i++) { console.log(i); }", options: [{allowForLoopAfterthoughts: true}] }
     ],
-
     invalid: [
-        { code: "var foo = 0; foo++;", errors: [{ message: "Unary operator '++' used.", type: "UpdateExpression"}] },
-        { code: "var foo = 0; foo--;", errors: [{ message: "Unary operator '--' used.", type: "UpdateExpression"}] },
-        { code: "for (i = 0; i < l; i++) { console.log(i); }", errors: [{ message: "Unary operator '++' used.", type: "UpdateExpression"}] },
+        {
+            code: "function foo() { return i++; }",
+            errors: [{ message: "Unary operator '++' used.", type: "UpdateExpression"}]
```

Comment:
Can you add an `output` property to this test? You can make it the same as the code:

```js
{
    code: "function foo() { return i++; }",
    output: "function foo() { return i++; }",
    errors: // ...
}
```

This will ensure that no fix is performed in this case.